pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [tool.poetry]
 name = "hathor-explorer-service"
 version = "0.18.2"
+package-mode = false
```

Comment:
With poetry 2 now we have to explicitly tell poetry that our project is not a package ([docs](https://python-poetry.org/docs/basic-usage/#operating-modes)).